src/app/business/block/file-share-detail/file-share-detail.component.ts

```diff
@@ -216,12 +216,16 @@ export class FileShareDetailComponent implements OnInit{
             str.forEach(item=>{
                 if(item.fileshareId == fileShareId){
                     item.createdAt = Utils.formatDate(item.createdAt);
+                    item.date = new Date(Utils.formatDate(item.createdAt)).getTime();
                     if(!item.description){
                         item.description = "--";
```

Comment:
I think this check can be in the view file in the html. No need to do this here.

Reply:
![K)8L5`MF9_CTD@_KUM3ZL0C](https://user-images.githubusercontent.com/37950702/61424332-b077f580-a945-11e9-8f10-64d8f60aa8f9.png)
but If we check in an HTML file like this, it won't work. Of course, I'm going to change the way I write it here, just to make it better.

Reply:
Oh! It is corresponding to the p-table field. Then it is alright.